rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt

```diff
@@ -1076,6 +1105,12 @@ open class Kotlin11Generator(
         println()
     }
 
+    private fun PrettyPrinter.toplevelExtensionTrait(decl: Ext) {
+        val extName = decl.extName ?: decl.name.decapitalize()
+        + """val IProtocol.$extName get() = getOrCreateExtension(${decl.name}::class) { ${decl.name}.createModel(lifetime, this) }"""
```

Comment:
Because this is not a block -- it is a one-line function and I implemented it exactly the same way as it is implemented for non-top-level extension in `Kotlin11Generator.extensionTrait`.

Reply:
I mean the  `{ ${decl.name}.createModel(lifetime, this) }` part

Reply:
But it's not critical indeed

Reply:
I don't see a reason why it is better for it to be a three-line block and not a one-line. But I can change it if it seems better.